glossary.md

```diff
@@ -1,16 +1,82 @@
 You may also need to check https://chromium.googlesource.com/chromiumos/docs/+/master/glossary.md.
 
-* LGTM: "Looks good to me", commonly used to approve a code review.
-* PTAL: Please take a look.
-* RSLGTM: "Rubber-stamp looks good to me". The reviewer approving without doing
+* ABI: [Application Binary Interface](https://en.wikipedia.org/wiki/Application_binary_interface)
+* ASAP: As Soon As Possible.
+* ASM: Assembler - General term for language translator, especially
```

Comment:
To clarify, I still think this criteria of inclusion in this list seems puzzling - if we are explaining what assembler is, why don't we also explain compiler, linker...and to that end, computer?

Reply:
@joyeecheung -

> why don't we also explain about compiler, linker...and to that end, computer?

Is this a joke, or sarcasm?

criteria of inclusion in this list is very simple IMO (I proposed majority of these items) - presence of their usage in our code base and that are not obvious to a user who are normally skilled in the art.

first criteria is objective, measurable:

https://github.com/nodejs/node/blob/c6ed01cf6d0d54f8de862fd41413e5cbac2a8ba4/configure.py#L744

second criteria is subjective, and my assertions are based on my interactions with users, mostly through the help repo: Most users there know what a computer is, but may not know what `ASM` is.

Reply:
>  warn('''No acceptable ASM compiler found! 

To me in this particular example it looks like the text should be changed to `assembler` instead, or something more informative like `.asm` compiler (I am not even sure if that's a proper term), instead of adding a glossary for it. `ASM compiler` AFAIK, is not widely used in this repo when people talk about things either.

Reply:
@joyeecheung, removed `ASM` from the document.